app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -512,12 +512,12 @@ private void initAmazonFactory() {
         }
     }
 
-    public void toggleCursor(boolean enabled) {
+    public void setCursorEnabled(boolean toEnable) {
```

Comment:
nit: `setIsCursorEnabled(boolean isEnabled)` is a little clearer but this is fine.